test/special.jl

```diff
@@ -22,17 +22,17 @@
     end
     function offset_exp!(A, B)
```

Comment:
I know I'm the one who named these...but in what world does `offset_exp*` make sense?